### PR TITLE
feat: ray on vertex - torch sample

### PR DESCRIPTION
**Pull request by inardini**
<br>
This tutorial demostrates how to use Ray on Vertex AI SDK and Vertex AI SDK for training and serving an PyTorch image classification model.
<br><br><br>

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [x] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [x] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

File: notebooks/official/ray_on_vertex_ai/get_started_with_pytorch_rov.ipynb [hunk 1]
```diff
@@ -0,0 +1,1646 @@
+{
```
Comment (gericdong) [inline notebooks/official/ray_on_vertex_ai/get_started_with_pytorch_rov.ipynb:1]: <p>demostrates &gt; demonstrates</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2711/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (gericdong) [inline notebooks/official/ray_on_vertex_ai/get_started_with_pytorch_rov.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #11.    ! pip3 install {USER} google-auth==2.27.0 etils==1.6.0 -q --no-warn-conflicts
</pre><p>If the version for etils is no significant, can we not pin the version or pin to one of these (1.5.2 or 1.5.1)</p><p> </p><pre class="ql-syntax" spellcheck="false">There was an error in the integration test: ERROR: No matching distribution found for etils==1.6.0
</pre><p> </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2711/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (gericdong) [inline notebooks/official/ray_on_vertex_ai/get_started_with_pytorch_rov.ipynb:1]: <p>enviroment &gt; environment</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2711/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Comment (inardini) [reply notebooks/official/ray_on_vertex_ai/get_started_with_pytorch_rov.ipynb:1]: done

Comment (inardini) [reply notebooks/official/ray_on_vertex_ai/get_started_with_pytorch_rov.ipynb:1]: done

Comment (inardini) [reply notebooks/official/ray_on_vertex_ai/get_started_with_pytorch_rov.ipynb:1]: done

**Review by gericdong — APPROVED** [review]

**Pull request merged**